src/const-eval.md

```diff
@@ -20,19 +20,23 @@ Additionally constant evaluation can be used to reduce the workload or binary
 size at runtime by precomputing complex operations at compiletime and only
 storing the result.
 
-Constant evaluation can be done by calling the `const_eval` query of `TyCtxt`.
+Constant evaluation can be done by calling the `const_eval_*` queries of `TyCtxt`.
 
-The `const_eval` query takes a [`ParamEnv`](./param_env.html) of environment in
-which the constant is evaluated (e.g. the function within which the constant is
-used) and a `GlobalId`. The `GlobalId` is made up of an
-`Instance` referring to a constant or static or of an
-`Instance` of a function and an index into the function's `Promoted` table.
+The `const_eval_*` queries use a [`ParamEnv`](./param_env.html) of environment
```

Comment:
I assume you mean `const_eval_poly` and friends? These are not queries, they are just wrappers around the `const_eval` query.

Reply:
Yeah, and I didn't know that. I updated this wording because someone was confused that `const_eval` doesn't exist on the docs anymore. Addressed via 852baf504063b8b4f522c2ebb5ca1fde2f1a105a.